tilings/algorithms/requirement_placement.py

```diff
@@ -377,15 +377,19 @@ def place_point_of_req(
                 if include_not:
                     res.append(self._tiling.__class__(obs + rem_req, reqs, ass))
                 continue
-
             forced_obs = self.forced_obstructions_from_requirement(
                 gps, indices, cell, direction
             )
+            forced_obs = [
+                o1
+                for o1 in forced_obs
+                if not any(o2 in o1 for o2 in filterfalse(o1.__eq__, forced_obs))
```

Comment:
Why is this different / better than
`if not any(o2 in o1 for o2 in forced_obs if o2 != o1)`
?